package.json

```diff
@@ -23,11 +23,14 @@
   },
   "devDependencies": {
     "@changesets/cli": "^2.29.6",
-    "@codama/renderers-js": "^1.3.4",
+    "@codama/renderers-js": "^1.3.5",
     "@iarna/toml": "^2.2.5",
+    "@metaplex-foundation/mpl-token-metadata": "^3.4.0",
     "@solana-program/system": "^0.8.0",
+    "@solana/spl-token": "^0.4.14",
```

Comment:
Let's remove the `@metaplex-foundation/mpl-token-metadata` and the `@solana/spl-token` dependencies. 🙏